src/index.ts

```diff
@@ -67,6 +67,7 @@ export default async function app(
     logger: loggerConfig,
     disableRequestLogging: true,
     trustProxy: true,
+    useSemicolonDelimiter: true,
```

Comment:
Not 100% if we use it, I think not, but leaving for BC just in case https://fastify.dev/docs/latest/Guides/Migration-Guide-V5/#usesemicolondelimiter-false-by-default